tests/functional/config.js

```diff
@@ -23,6 +23,9 @@ let config = {
     components: [
       `${paths.test.wdioSpec}/components/*.spec.js`
     ],
+    features: [
+      `${paths.test.wdioSpec}/features/**/*.spec.js`
```

Comment:
given its all in the one feature spec now, shall we do what we did with components and just have it at feature level instead of in a sub folder

Reply:
Scrap this, i've realised its number within routing, makes sense this way